src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/ExtensionMethodImportCompletionHelper.cs

```diff
@@ -89,22 +89,30 @@ private static ImmutableArray<string> GetReceiverTypeNames(ITypeSymbol receiverT
             return allTypeNamesBuilder.ToImmutableArray();
         }
 
-        private static string GetReceiverTypeName(ITypeSymbol receiverTypeSymbol)
+        private static string GetReceiverTypeName(ITypeSymbol typeSymbol)
         {
-            if (receiverTypeSymbol is IArrayTypeSymbol arrayTypeSymbol)
+            switch (typeSymbol)
             {
-                var elementTypeSymbol = arrayTypeSymbol.ElementType;
-                while (elementTypeSymbol is IArrayTypeSymbol symbol)
-                {
-                    elementTypeSymbol = symbol.ElementType;
-                }
+                case INamedTypeSymbol namedType:
+                    return namedType.IsTupleType ? string.Empty : namedType.MetadataName;
```

Comment:
do we have tests for these special cases?

Reply:
This is actually incorrect. I have fixed the handling of ValueTuple type in source and added tests.